ncm-network/src/main/pan/components/network/core-schema.pan

```diff
@@ -125,6 +125,9 @@ type structure_interface = {
     "stp" ? boolean # enable/disable stp on bridge (true: STP=on)
     "delay" ? long # brctl setfd DELAY
     "bridging_opts" ? structure_bridging_options
+
+    "devicetype" ? string with match(SELF, '^(ovs)$')
+    "ovs_bridge" ? string with exists ("/system/network/interfaces/" + SELF)
 };
```

Comment:
I cannot find an example with two conditions checks in schema. Do you have some examples?